libs/subliminal_patch/providers/subssabbz.py

```diff
@@ -76,6 +78,14 @@ def get_matches(self, video):
              matches.add('hash')
 
         matches |= guess_matches(video, guessit(self.filename, {'type': self.type}))
+
+        if self.fps and video.fps and float(self.fps) != float(video.fps):
+            logger.info('FPS do not match for %r, %s vs %s. Reducing score.', self.page_link, self.fps, video.fps)
+            return set([x for x in matches if x in ['series', 'season', 'episode']])
+        else:
+            matches.add('fps')
+            matches.add('format')
```

Comment:
Don't think there is a 'fps' match yet and why do you set 'format' if case of fps match?

Reply:
`matches.add('fps')` will make "fps" to be shown in the manual search UI, but in reality it doesn't change the overall score as it doesn't have an assigned value - see https://github.com/morpheus65535/bazarr/blob/development/libs/subliminal/score.py#L40

The goal is to use `format` to actually bump up the score as `format` has no other meaning than calculating the score, we (vs others, where the absence of the match flag will reject the subtitle - see https://github.com/morpheus65535/bazarr/blob/development/bazarr/get_subtitle.py#L339)

Reply:
That's fine for me.

Reply:
So do you think that HDTV, DVD, BluRay or WEB-DL are all the same if their frame rate is the same? By setting matches.add('format') you are giving them same score result.